### DISCUSSION TITLE
Multiple Builds

### DISCUSSION THREAD

**Discussion by ajowsey** [nrobinson2000/neopo#21] [Q&A] [answered]
@nrobinson2000 Can neopo build two different directories at the same time?

Obviously the windows/vscode particle workbench cant do this. But can neopo?

**Comment by nrobinson2000** [comment, marked as answer]
@ajowsey 

You could achieve this by creating a neopo install in each project directory and using the NEOPO_PATH environment variable.

Create two example projects and install neopo in them:
```
cd /tmp

neopo create project1
NEOPO_PARALLEL=1 NEOPO_PATH="project1/neopo" neopo install

neopo create project2
NEOPO_PARALLEL=1 NEOPO_PATH="project2/neopo" neopo install
```

Build the two projects simultaneously:


Terminal 1:

```
cd /tmp/project1
export NEOPO_PATH="$PWD/neopo"
time neopo build -v
```

Terminal 2:
```
cd /tmp/project2
export NEOPO_PATH="$PWD/neopo"
time neopo build -v
```
